### ISSUE TITLE
DLITE comparison

### ISSUE THREAD

**Issue by JerryLi4533** [keweimao/DeepDelight#6]
I have used dlite as loss function to train the gpt model. I tried to visualize the loss with line plot and the result seems not very well. Hope we can find a better way to implement loss function so we can have higher training efficiency and accuracy. The notebook is added to Thread 1 folder.

**Comment by lixiao-yang** [comment]
I think maybe you still need to evaluate DLITE loss when you use cross entropy as the loss function since only optimize cross entropy may cause different results than evaluating them both in each step. However, I got the same result as yours so the general conclusion is the same.
